provision/virtualbox.sh

```diff
@@ -3,6 +3,4 @@
 set -e
 set -x
 
-sudo pacman -S --noconfirm linux-headers
-sudo pacman -S --noconfirm virtualbox-guest-utils-nox
-sudo systemctl enable vboxservice
```

Comment:
`VirtualBox can also synchronize the time between the host and the guest, to do this, start/enable the vboxservice.service.`